tests/bgp/test_bgp_dual_asn.py

```diff
@@ -252,14 +256,17 @@ def dual_asn_setup(
             )
 
         ptfhost.shell("ip route flush %s/%d" % (self.lo["addr"], self.lo["prefixlen"]))
-        ptfhost.shell(
-            "ip route add %s/%d via %s"
-            % (self.lo["addr"], self.lo["prefixlen"], vlan_addr)
-        )
-        ptfhost.shell(
-            "ip -6 route add %s/%d via %s"
-            % (self.lo6["addr"], self.lo6["prefixlen"], vlan_addr6)
-        )
+        try:
+            ptfhost.shell(
+                "ip route add %s/%d via %s"
+                % (self.lo["addr"], self.lo["prefixlen"], vlan_addr)
+            )
+            ptfhost.shell(
+                "ip -6 route add %s/%d via %s"
+                % (self.lo6["addr"], self.lo6["prefixlen"], vlan_addr6)
+            )
+        except Exception:
+            logger.info("route may already exists, ignore error !")
```

Comment:
@jcaiMR  should we try to check the existence of the route?

Reply:
My thinking no need, here ip route add command only return error when route already exists, because nexthop (via vlan addr) always there (we get from interface).